src/store.ts

```diff
@@ -289,6 +322,7 @@ class Store extends ElectronStore<StoreData> {
                 if (safeStorageBackend === "basic_text") {
                     return this.upgradeLinuxBackend1();
                 } else if (safeStorageBackend === "plaintext") {
+                    this.secrets = new PlaintextStorageWriter(this);
```

Comment:
Should this maybe go into the method so the `if (!this.secrets) throw new Error` can go away?

Reply:
https://github.com/element-hq/element-desktop/pull/2338/commits/63e1e0d894ddb19bf48c444ebfd7f0e9abbd3c06